grabbit/src/main/groovy/com/twcable/grabbit/ClientJobStatus.groovy

```diff
@@ -82,4 +100,12 @@ class ClientJobStatus {
             get(explorer, id)
         }
     }
+
+    static List<ClientJobStatus> getAllForTransaction(@Nonnull final JobExplorer explorer, @Nonnull final long transactionID) {
```

Comment:
What does this return if the `transactionID` never existed? What if it did exist, but all of its jobs are complete? What if it did exist but it was a long time ago and potentially pruned?